### PR TITLE
8th and final round of migrating integration tests from Python

### PR DESCRIPTION
**Pull request by robsdedude**
https://github.com/neo4j/neo4j-python-driver/pull/873

### CODE REVIEW

File: tests/stub/session_run_parameters/test_session_run_parameters.py [hunk 1]
```diff
@@ -116,7 +116,9 @@ def test_tx_meta(self):
             with self.subTest(routing=routing):
                 self._run("tx_meta", routing,
                           session_args=("w",),
-                          run_kwargs={"tx_meta": {"akey": "aval"}})
+                          run_kwargs={
+                              "tx_meta": {"akey": types.CypherString("aval")}
```
Comment (injectives) [inline tests/stub/session_run_parameters/test_session_run_parameters.py:120]: If we are changing to supplying values as explicit Cypher values, we might want to update it in other places as well. For instance, Java backend does not expect this.

An example of other place: https://github.com/neo4j-drivers/testkit/blob/5.0/tests/stub/driver_parameters/test_bookmark_manager.py#L120

Comment (robsdedude) [reply tests/stub/session_run_parameters/test_session_run_parameters.py:120]: Absolutely! I missed that one, thanks for spotting.

File: nutkit/frontend/session.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ def close(self, hooks=None):
         if not isinstance(res, protocol.Session):
             raise Exception("Should be session but was: %s" % res)
 
-    def run(self, cypher, params=None, tx_meta=None, hooks=None, **kwargs):
+    def run(self, cypher, *, params=None, tx_meta=None, hooks=None, **kwargs):
```
Comment (bigmontz) [inline nutkit/frontend/session.py:19]: Does this `*` mean `extra positional params i don't care`?

**Review by bigmontz — APPROVED** [review]
❗

File: nutkit/frontend/session.py (hunk 2 above)
Comment (robsdedude) [reply nutkit/frontend/session.py:19]: Almost. `*args` (or any other name after `*`) means `collect extra positional arguments in that variable`. If you just write `*` it means from here on, only keyword arguments. FYI: With Python 3.8 they also introduced `/` wich means up until here, positional arguments only ([the PEP](https://peps.python.org/pep-0570/) explaining it).

**Pull request merged**